src/review/forms.py

```diff
@@ -209,8 +209,12 @@ class FakeReviewerDecisionForm(FakeModelForm, ReviewerDecisionForm):
 
     def __init__(self, *args, **kwargs):
         kwargs["disable_fields"] = True
+        open_peer_review = kwargs.pop('open_peer_review')
```

Comment:
should provide a default to kwargs.pop() to avoid a `KeyError`, eg:

`open_peer_review = kwargs.pop('open_peer_review', None)`